wiki/Play_style/en.md

```diff
@@ -6,7 +6,7 @@ This page will detail the various ways in which players can choose to play in th
 
 ### Mouse-only
 
-*Note: Mouse-only was once the only way to play osu! until other playing methods were supported over time.*
+Mouse-only was once the *only* way to play osu! until other playing methods were supported over time.
```

Comment:
This sentence is used in similar ways throughout the article, but every other version doesn't have a note.